base/logging.cc

```diff
@@ -478,9 +478,15 @@ SbLogPriority LogLevelToStarboardLogPriority(int level) {
     case LOG_ERROR:
       return kSbLogPriorityError;
     case LOG_FATAL:
-    case LOG_VERBOSE:
       return kSbLogPriorityFatal;
+    case LOG_VERBOSE:
     default:
+      if (level <= LOG_VERBOSE) {
+        // Verbose level can be any negative integer, sanity check its range to
```

Comment:
Why do we need to sanity check the level?
What happens if we don't?

Reply:
I am not sure if this code will be used in another place in future (as it's Chromium code, not Cobalt code), where it may silently take an integer unintentionally set to a very small number (or overflowed).  This will catch such kind of issue, and can be easily refined if the number is intentional.

Reply:
ok, sg

Reply:
Thank you Yavor!